kubernetes_deploy/development/queue_worker_deployment.yml

```diff
@@ -0,0 +1,62 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  annotations:
+    kubernetes.io/change-cause: "<to be filled in deploy_to_kubernetes script>"
+  name: laa-legal-adviser-api-worker
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        worker: laa-legal-adviser-api-worker
+        env: staging
+        service_area: laa-get-access
+        service_team: cla-fala
+    spec:
+      containers:
+      - image: "<to be set by deploy_to_kubernetes>"
+        name: worker
+        args: ["/home/app/docker/run_worker.sh"]
+        env:
+        - name: WORKER_APP_CONCURRENCY
+          value: "2"
+        - name: LOG_LEVEL
+          value: INFO
+        - name: SECRET_KEY
+          valueFrom:
+            secretKeyRef:
+              name: secret-key
+              key: SECRET_KEY
+        - name: SENTRY_DSN
+          valueFrom:
+            secretKeyRef:
+              name: sentry
+              key: dsn
+        - name: DB_USERNAME
+          valueFrom:
+            secretKeyRef:
+              name: db
+              key: user
+        - name: DB_NAME
+          valueFrom:
+            secretKeyRef:
+              name: db
+              key: name
+        - name: DB_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              name: db
+              key: password
+        - name: DB_HOST
+          valueFrom:
+            secretKeyRef:
+              name: db
+              key: host
+        - name: DB_PORT
+          valueFrom:
+            secretKeyRef:
+              name: db
+              key: port
+        - name: RABBITMQ_HOST
+          value: mq
```

Comment:
Can be replaced with
```yaml
- name: AMQP_BROKER_URL
  value: "amqp://guest:guest@localhost:5672//"
```

Reply:
`- name: AMQP_BROKER_URL
  value: "amqp://guest:guest@laa-legal-adviser-api-mq:5672//"`

As the workers are in another pod